docs/pages/versions/unversioned/sdk/location.mdx

```diff
@@ -100,6 +100,17 @@ Learn how to configure the native projects in the [installation instructions in
 
 </ConfigReactNative>
 
+### Background location
+
+Background location allows your app to receive location updates while it is running in the background and includes both location updates and region monitoring through geofencing. This feature is subject to platform API limitations and system constraints:
+
+- Background location will stop if the user terminates the app.
+- Background location resume if the user restarts the app.
+- On Android, a terminated app will not automatically restart when a location or geofencing event occurs due to platform limitations. <PlatformTag platform="android" />
+- On iOS, terminated apps will be restarted by the system when new geofence event occurs. <PlatformTag platform="ios" />
```

Comment:
```suggestion
- On iOS, the system will restart terminated apps when a new geofence event occurs. <PlatformTag platform="ios" />
```

Missing article and we can also avoid passive voice usage in the first half of the sentence.